Python/Setup/swix/Microsoft.PythonTools.vsmanproj

```diff
@@ -10,13 +10,11 @@
 
     <BuildNumber>$(VSTarget).$(BuildNumber)</BuildNumber>
 
-    <TargetName>Microsoft.PythonTools</TargetName>
-    <OutputPath>$(SetupOutputPath)swix\</OutputPath>
-    <OutDir>$(OutputPath)</OutDir>
+    <TargetName>$(MSBuildProjectName)</TargetName>
   </PropertyGroup>
 
   <ItemGroup>
-    <MergeManifest Include="$(OutputPath)*.json" />
+    <MergeManifest Include="$(OutputPath)Microsoft.*.json" />
```

Comment:
But since we currently have Microsoft.VisualStudio.Workload.Python in this repo, it can't be `Microsoft.PythonTools.*.json` like I intended. But maybe we'll move the workload back into the VS repo at some point... not sure yet.

---
In reply to: [69643277](https://github.com/Microsoft/PTVS/pull/1396#discussion_r69643277) [](ancestors = 69643277,69643035)